test/unit/spec/narrative_core/upload/importSetup-spec.js

```diff
@@ -282,7 +290,27 @@ define([
             });
 
             // TODO
-            xit('should error properly when unable to find bulk specification info', async () => {});
+            it('should error properly when unable to find bulk specification info', async () => {
```

Comment:
is the TODO label still required?

Reply:
Nope, but the next one is! There's another case that should be tested in it that isn't fleshed out yet - will be in the next PR.